### PR TITLE
Add a Sites e2e test

### PR DESCRIPTION
**Pull request by sbelsk**
This PR adds a new end-to-end test to the `/sites` page, and removes some dead selenium test code related to this same page.

### CODE REVIEW

File: tests/cypress/e2e/sites.cy.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+describe('site statistics', () => {
+  it('is proected by login', () => {
```
Comment (williamjallen) [inline tests/cypress/e2e/sites.cy.js:2]:
```suggestion
  it('is protected by login', () => {
```

File: tests/cypress/e2e/sites.cy.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      .and('contain', 'Total Physical Memory:')
+      .and('contain', 'Description:')
+      .and('contain', 'This site belongs to the following projects:');
+    cy.get('#main_content').find('a').each(project_url => {
+      cy.wrap(project_url).should('have.attr', 'href').and('contain', 'index.php?project=');
+    });
+  });
+
+  it('loads the "Time spent" graph', () => {
+    cy.visit('sites/6');
```
Comment (williamjallen) [inline tests/cypress/e2e/sites.cy.js:54]: Is there a way to get the ID dynamically?

**Review by williamjallen — CHANGES_REQUESTED** [review]

File: tests/cypress/e2e/sites.cy.js (hunk 1 above)
Comment (sbelsk) [reply tests/cypress/e2e/sites.cy.js:2]: Good catch.

File: tests/cypress/e2e/sites.cy.js (hunk 2 above)
Comment (sbelsk) [reply tests/cypress/e2e/sites.cy.js:54]: Sure, I'll navigate there instead.

**Review by williamjallen — APPROVED** [review]

**Pull request merged**